psi4/driver/procrouting/sapt/sapt_proc.py

```diff
@@ -386,13 +386,36 @@ def sapt_dft(dimer_wfn, wfn_A, wfn_B, sapt_jk=None, sapt_jk_B=None, data=None, p
     if cleanup_jk:
         sapt_jk.finalize()
 
+    # Hybrid xc kernel check
+    do_hybrid = core.get_option("SAPT", "SAPT_DFT_DO_HYBRID")
+    is_x_hybrid = wfn_B.functional().is_x_hybrid()
+    is_x_lrc = wfn_B.functional().is_x_lrc()
+    hybrid_specified = core.has_option_changed("SAPT", "SAPT_DFT_DO_HYBRID")
+    if is_x_lrc:
+        if do_hybrid:
+            if hybrid_specified:
+                raise ValidationError("SAPT(DFT): Hybrid xc kernel not yet implemented for range-separated funtionals.")
+            else:
+                core.print_out("Warning: Hybrid xc kernel not yet implemented for range-separated funtionals; hybrid kernel capability is turned off.\n") 
```

Comment:
note for other reviewers: this logic is equivalent to the lower https://github.com/psi4/psi4/pull/1865#discussion_r433598188 table except the "does nothing"+"lrc" is a written warning, not an error. This isn't too nice for the users. If you like I can put in an UpgradeHelper that errors out and nudges them toward an explicit do_hybrid=False input file.

Reply:
`do_hybrid` for LDA/GGA should just silently work I think. There is no reason to raise an error for it (IMO).

Reply:
> `do_hybrid` for LDA/GGA should just silently work I think. There is no reason to raise an error for it (IMO).

Will silently work if not specified. Error only raised when do_hybrid manually set to true with LDA/GGA.